test/svm/SvmSpoke.common.ts

```diff
@@ -22,9 +22,9 @@ const exclusiveRelayer = Keypair.generate().publicKey;
 const outputToken = new PublicKey("1111111111113EsMD5n1VA94D2fALdb1SAKLam8j"); // TODO: this is lazy. this is cast USDC from Eth mainnet.
 const inputAmount = new BN(500000);
 const outputAmount = inputAmount;
-const quoteTimestamp = new BN(Math.floor(Date.now() / 1000) - 10); // 10 seconds ago.
+const quoteTimestamp = new BN(Math.floor(Date.now() / 1000) - 50); // 10 seconds ago.
 const fillDeadline = new BN(Math.floor(Date.now() / 1000) + 600); // 600 seconds from now.
-const exclusivityDeadline = new BN(Math.floor(Date.now() / 1000) + 300); // 300 seconds from now.
+const exclusivityPeriod = new BN(300); // 300 seconds.
```

Comment:
this update applies to match the EVM changes and the correct input prop and how it's meant to be used.